cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/service/DatasetInstanceService.java

```diff
@@ -472,12 +477,8 @@ private void disableExplore(DatasetId datasetInstance) {
     try {
       exploreFacade.disableExploreDataset(datasetInstance.toId());
     } catch (Exception e) {
-      String msg = String.format("Cannot disable exploration of dataset instance %s: %s",
-                                 datasetInstance, e.getMessage());
-      LOG.error(msg, e);
+      LOG.error("Cannot disable explore for dataset instance {}: {}", datasetInstance, e.getMessage(), e);
```

Comment:
That was the case before, too. I think the intention was that eventually we will throw an exception with that message, instead of logging and ignoring it. Then calling getMessage() on that exception would return the entire message including the cause, and that is all we need to display to the client. But because unfortunately, we have too many Throwables.propagate() in our code, the true cause if often nested deeper. I believe that is the reason why the existing code was logging both the message and the exception. I just kept it that way.